pyat/at/lattice/elements.py

```diff
@@ -473,3 +457,14 @@ def __init__(self, family_name, length, kick_angle, **kwargs):
         super(Corrector, self).__init__(family_name,
                                         Length=kwargs.pop('Length', length),
                                         KickAngle=kick_angle, **kwargs)
+
+
+class _RingParam(Element):
+    """Private class for Matlab RingParam element"""
```

Comment:
It might be helpful to explain why this class is private.

Reply:
It's now removed from elements and is a private class in` at.load.utils`.